app/controllers/gardens_controller.rb

```diff
@@ -4,7 +4,9 @@ class GardensController < DataController
   def index
     @owner = Member.find_by(slug: params[:member_slug])
     @show_all = params[:all] == '1'
+    @my_garden = params[:member_slug].present? ? true : false
```

Comment:
this variable name is confusing. how about 
```suggestion
    @show_member = params[:member_slug].present? ? true : false
```

Reply:
I agree! Suggested term can be also read in 2 different ways - to show a member or that is a show action for member. How about `show_jump_to` - because that is why it is ultimately used for.